include/oxen/quic/address.hpp

```diff
@@ -390,4 +398,15 @@ namespace std
             return h;
         }
     };
+
+    template <>
+    struct hash<oxen::quic::Path>
+    {
+        size_t operator()(const oxen::quic::Path& addr) const
+        {
+            auto h = hash<oxen::quic::Address>{}(addr.local);
+            h ^= hash<oxen::quic::Address>{}(addr.remote);
```

Comment:
See the hash combining just above this using a quasi-random value (`inverse_golden_ratio`) and some bit shifting.  You want something like that when combining hashes so that correlation of the two values generally doesn't result in correlation of the hash.  With just a plain xor you get these undesirable characteristics:
```C++
Address a = /* anything */;
Address b = /* something else */;
hash(Path{a, b}) == hash(Path{b, a})
hash(Path{a, a}) == hash(Path{b, b}) == 0
```
but using the same `h ^= hash<oxen::quic::Address>{}(addr.remote) + inverse_golden_ratio + (h << 6) + (h >> 2);` here avoids such correlations.